purescript.cabal

```diff
@@ -233,6 +233,7 @@ library
     Language.PureScript.CodeGen.JS
     Language.PureScript.CodeGen.JS.Common
     Language.PureScript.CodeGen.JS.Printer
+    Language.PureScript.CodeGen.JS.CoreFnOptimizer
```

Comment:
While we're on the pedantry train, please keep this list sorted.

Reply:
@rhendric Can you please review these changes and approve this PR, if possible ?